docs/content/v0.20/examples.md

```diff
@@ -52,17 +52,32 @@ which installs OCM on it.
 
    see [here](./architecture.md#ocm-status-add-on-agent) for more details on the add-on.
 
-5. Create a Workload Description Space `wds1` in KubeFlex. Similarly to before, `-p kubestellar`
+1. Create a Workload Description Space `wds1` in KubeFlex. Similarly to before, `-p kubestellar`
 runs a post-create hook on the *k8s* control plane that starts an instance of a KubeStellar controller
 manager which connects to the `wds1` front-end and the `imbs1` OCM control plane back-end.
 
    ```shell
    kflex create wds1 -p kubestellar
    ```
 
-6. Follow the steps to [create and register two clusters with OCM](example-wecs.md).
+1. Build and run the OCM based transport controller as executable process.  
+**NOTE**: This is work in progress and we're working on running this controller in a pod.
 
-7. (optional) Check relevant deployments and statefulsets running in the hosting cluster. Expect to
+   OCM based transport controller is built in a different package, so we need to download the package and run the executable as background process.
```

Comment:
But you don't "download executable"  you download the source and rebuilt it.

Reply:
you are correct, fixed.